workflow/controller/dag.go

```diff
@@ -3,11 +3,12 @@ package controller
 import (
 	"encoding/json"
 	"fmt"
```

Comment:
Is your format is correct ?

Reply:
I applied both gofmt and goimports and checked with the linter.